NDB.Covid19/NDB.Covid19.Droid/Utils/ForegroundServiceHelper.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Android.App;
+using Android.Content;
+using Android.OS;
+using Android.Runtime;
+using Android.Views;
+using Android.Widget;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace NDB.Covid19.Droid.Utils
+{
+    public class ForegroundServiceHelper
+    {
+        public static void StartForegroundServiceCompat<T>(Context context, Bundle args = null) where T : Service
+        {
+            Intent intent = new Intent(context, typeof(T));
+            if (args != null)
+            {
+                intent.PutExtras(args);
+            }
+
+            if (Build.VERSION.SdkInt >= BuildVersionCodes.O)
+            {
+                //context.StartForegroundService(intent);
+                context.StartService(intent);
```

Comment:
why you have changed it to StartService?

Reply:
I got an error that StartForegroundService() did not then call Service.startForeground() and with use of StartService it worked ok.

Reply:
do not do this. StartService starts the service in the background and service may be killed by power manager after few minutes so user will never see the reminder.

Read https://stackoverflow.com/questions/44425584/context-startforegroundservice-did-not-then-call-service-startforeground to solve startForeground issue

Reply:
Ok I will change it